### PR TITLE
EUI-5590 - Specific Access Request approved

### PR DESCRIPTION
**Pull request by turner1979**
### JIRA link ###

https://tools.hmcts.net/jira/browse/EUI-5590

### Change description ###

* UI for specific access request approved screen

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/role-access/containers/specific-access/specific-access-approved/specific-access-approved.component.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.govuk-grid-column-two-thirds {
+  padding-left: 0;
```
Comment (paulhowes-HMCTS) [inline src/role-access/containers/specific-access/specific-access-approved/specific-access-approved.component.scss:2]: I don't think you should be overriding the GDS standards. What is causing the padding?

File: src/role-access/containers/specific-access/specific-access-approved/specific-access-approved.component.ts [hunk 2]
```diff
@@ -7,7 +7,8 @@ import * as fromFeature from '../../../store';
 
 @Component({
   selector: 'exui-specific-access-approved',
-  templateUrl: './specific-access-approved.component.html'
+  templateUrl: './specific-access-approved.component.html',
+  styleUrls: ['./specific-access-approved.component.scss']
```
Comment (paulhowes-HMCTS) [inline src/role-access/containers/specific-access/specific-access-approved/specific-access-approved.component.ts:11]: This should not be needed if you eliminate the padding issue

**Review by paulhowes-HMCTS — CHANGES_REQUESTED** [review]
One minor request for change as you shouldn't have to override the GDS guidelines.

File: src/role-access/containers/specific-access/specific-access-approved/specific-access-approved.component.scss (hunk 1 above)
Comment (turner1979) [reply src/role-access/containers/specific-access/specific-access-approved/specific-access-approved.component.scss]: @paulhowes-HMCTS have removed original css rule added and used `govuk-!-padding-left-0` class as discussed

File: src/role-access/containers/specific-access/specific-access-approved/specific-access-approved.component.ts (hunk 2 above)
Comment (turner1979) [reply src/role-access/containers/specific-access/specific-access-approved/specific-access-approved.component.ts]: have removed

**Review by paulhowes-HMCTS — APPROVED** [review]
Approving after change to remove unnecessary css implemented.

**Review by tarikdogan-hmcts — APPROVED** [review]

**Review by udaydenduluri33 — APPROVED** [review]
@turner1979 thanks for the changes

**Pull request merged**